relayer/relayer.go

```diff
@@ -174,7 +169,7 @@ func NewRelayer(
 		// ProcessFromHeight to overload the message queue and cause a deadlock.
 		go sub.ProcessFromHeight(big.NewInt(0).SetUint64(height), r.catchUpResultChan)
 	} else {
-		err = r.setProcessedBlockHeightToLatest()
+		_, err = r.setProcessedBlockHeightToLatest()
```

Comment:
Can we add a log here that we're initializing the relayer from the latest height?